load-test/stomp/.env

```
@@ -0,0 +1,2 @@
+LOAD_TEST_TOKEN = eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiIxIiwiYXV0aCI6IlJPTEVfVVNFUiIsImV4cCI6MTczODYwMzIzN30.2BQZ2P8qooLUeuE6Nl1RyBMpdAoJFum5ChfFK5l_eaY
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)